.github/workflows/ci.yml

```diff
@@ -69,19 +65,17 @@ jobs:
         run: composer install --prefer-dist --no-progress
 
       - name: Run phpstan
-        run: make phpstan
+        run: just lint
 
   tests:
-    name: Tests
-
     runs-on: ubuntu-latest
 
     strategy:
       fail-fast: false
       matrix:
-        env:
```

Comment:
this never worked as intended: `env` doesn't actually set the environment, it just sets a github variable called `env`